Makefile

```diff
@@ -105,16 +105,8 @@ driver-deps:
 	echo "Warning: kernel-header were not installed") \
 	&& echo "Successfully installed the driver deps"
 
-# In order to avoid executing the same rule everytime,
-# the build rules are prefixed by dot and are generating
-# a file with the same name via the touch command. This
-# change is required in order to capture the timestamp
-# of the rule.
-.build-container: tools/Dockerfile1804.${HOST_MACHINE}
-	docker image build -t $(CONTAINER_TAG) -f tools/Dockerfile1804.${HOST_MACHINE} tools/
-	touch $@
-
-build-container: .build-container
+build-container:
+	@docker image build --pull --quiet -t $(CONTAINER_TAG) -f tools/Dockerfile1804.${HOST_MACHINE} tools/
```

Comment:
won't it be too long to rebuild the container every time?

Reply:
It shouldn't build every time, it only builds if it's not in the cache, or if Dockerfile has changed or if the base image has changed.